modules/util/portal-tools-sample-sql-builder/src/main/resources/com/liferay/portal/tools/sample/sql/builder/dependencies/system_object_definitions.ftl

```diff
@@ -4,22 +4,22 @@
 
 ${dataFactory.toInsertSQL(objectFolderModel)}
 
-<#list dataFactory.newSystemObjectDefinitionModels(objectFolderModel.getObjectFolderId()) as systemObjectDefinitionModel>
-	${dataFactory.toInsertSQL(systemObjectDefinitionModel)}
+<#list dataFactory.newObjectDefinitionModels(objectFolderModel.getObjectFolderId()) as objectDefinitionModel>
+	${dataFactory.toInsertSQL(objectDefinitionModel)}
 
-	<#list dataFactory.newResourcePermissionModels(systemObjectDefinitionModel) as resourcePermissionModel>
+	<#list dataFactory.newResourcePermissionModels(objectDefinitionModel) as resourcePermissionModel>
 		${dataFactory.toInsertSQL(resourcePermissionModel)}
 	</#list>
 
-	<#list dataFactory.newObjectFieldModels(systemObjectDefinitionModel.getObjectDefinitionId(), systemObjectDefinitionModel.getDBTableName(), 0, systemObjectDefinitionModel.getPKObjectFieldName()) as objectFieldModel>
+	<#list dataFactory.newObjectFieldModels(objectDefinitionModel.getObjectDefinitionId(), objectDefinitionModel.getDBTableName(), 0, objectDefinitionModel.getPKObjectFieldName()) as objectFieldModel>
 		${dataFactory.toInsertSQL(objectFieldModel)}
 	</#list>
 
-	<#if systemObjectDefinitionModel.getDBTableName() == "CommerceOrder">
-		 ${dataFactory.toInsertSQL(dataFactory.newObjectActionModel(systemObjectDefinitionModel.objectDefinitionId, notificationTemplateModel.notificationTemplateId))}
+	<#if objectDefinitionModel.getDBTableName() == "CommerceOrder">
+		 ${dataFactory.toInsertSQL(dataFactory.newObjectActionModel(objectDefinitionModel.objectDefinitionId, notificationTemplateModel.notificationTemplateId))}
 	</#if>
 
-	<#if systemObjectDefinitionModel.getDBTableName() == "User_">
-		${dataFactory.getExtensionDynamicObjectDefinitionTableCreateSQL(systemObjectDefinitionModel)}
+	<#if objectDefinitionModel.getDBTableName() == "User_">
+		${dataFactory.getExtensionDynamicObjectDefinitionTableCreateSQL(objectDefinitionModel)}
```

Comment:
@slnn Please also consider renaming these methods (create and insert). Make the name reflect the table name.

Reply:
Hi @dantewang , the method `getExtensionDynamicObjectDefinitionTableCreateSQL` is used to dynamically create a table for the object definition. It's being used for more than one object definition. So the name of the table varies.

Reply:
hi @dantewang , 

I'm not sure that I understand your point. But if I can contribute, the method `getExtensionDynamicObjectDefinitionTableCreateSQL` will generate the create SQL for a dynamic object definition table, so table name will vary